platforms/core-configuration/declarative-dsl-core/src/test/kotlin/org/gradle/internal/declarativedsl/dom/operations/overlay/DocumentOverlayTest.kt

```diff
@@ -229,7 +229,7 @@ class DocumentOverlayTest {
                 a = 33
                 b = 6
                 unresolved5()
-                errorExample = unsupported.propertyAccess
+                c = something.propertyAccess
```

Comment:
According to my other comment, this should still be represented by an unsupported syntax item in the DOM. Sorry that I was not clear enough from the start.